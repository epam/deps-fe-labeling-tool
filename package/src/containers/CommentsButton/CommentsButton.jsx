import React from 'react'
import { Button } from '@/components/Button'
import { CommentIcon } from '@/components/Icons/CommentIcon'
import { Tooltip } from '@/components/Tooltip'

const RESOURCE_TOOLTIP_COMMENTS = 'Comments'

const CommentsButton = () => (
  <Tooltip title={RESOURCE_TOOLTIP_COMMENTS}>
    <Button.Icon icon={<CommentIcon />} />
  </Tooltip>
)

export {
  CommentsButton
}
