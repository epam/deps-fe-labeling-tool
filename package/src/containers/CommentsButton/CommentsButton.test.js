import React from 'react'
import { shallow } from 'enzyme'
import { CommentsButton } from '@/containers/CommentsButton'

describe('Container: CommentButton', () => {
  const wrapper = shallow(<CommentsButton />)

  it('should render correct layout', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
