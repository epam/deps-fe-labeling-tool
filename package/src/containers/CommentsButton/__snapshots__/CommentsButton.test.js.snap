// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container: CommentButton should render correct layout 1`] = `
<Tooltip
  title="Comments"
>
  <IconButton
    icon={<CommentIcon />}
  />
</Tooltip>
`;
